### PR TITLE
Maurice Bokanga hw01 submission

### PR DESCRIPTION
**Pull request by mbokanga**
Maurice Bokanga's hw01 submission

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,25 @@
-# hw01
```
Comment (WuyunpengChen) [inline README.md:1]: Evaluation HW-1 for Maurice Bokanga
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     √      |              |            |
| **Coding strategy**         |     √      |              |            |
| **Presentation**            |    √       |              |            |
| **Achievement, creativity** |     √      |              |            |
| **Ease of access**          |    √       |              |            |

As this is the first assignment, there are not many comments I could make on this one.
The code was written in a proper way and it is quite interesting to see the bear appear before everything.
Overall it is a well-written code and obviously reached the given requirement.

**Comment by jingkunw** [comment]
Topic | Excellent | Satisfactory | Needs Work
-- | -- | -- | --
Coding style | √ |   |  
Coding strategy | √ |   |  
Presentation | √ |   |  
Achievement, creativity | √ |   |  
Ease of access | √ |   |

**Review by xiangyum — COMMENTED** [review]
Evaluation HW-xx for Jane Doe
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |        X   |              |            |
| **Coding strategy**         |     X      |              |            |
| **Presentation**            |       X    |              |            |
| **Achievement, creativity** |      X     |              |            |
| **Ease of access**          |     X      |              |            |

Remarks:
Creative and funny. Neat code, would definitely read again. Great grasp of Markdown.

**Comment by cernhofer** [comment]
Evaluation HW-01 for mbokanga  
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |       x    |              |            |
| **Coding strategy**         |      x     |              |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |     x      |              |            |
| **Ease of access**          |           |       x       |            |

Remarks:
* Great work! You included lots of interesting information and markdown elements.
* Make sure to include your progress/reflections in the comment of the pull request.

**Pull request closed without merging**